### DISCUSSION TITLE
Request: Support "countdown" / "duration" mode

### DISCUSSION THREAD

**Discussion by mars-lan** [henninghall/react-native-date-picker#199] [Ideas]
It's probably a big ask but is there any plan to support the "countdown" mode for both Android & iOS?

**Comment by henninghall** [comment]
It would be great to be able to support countdown. There is no concrete plan for it atm but I fully support the idea. If you want to start working on it, it is very appreciated!

**Comment by gaedlld** [comment]
Any news ??
